src/main/java/net/dv8tion/jda/api/interactions/callbacks/IMessageEditCallback.java

```diff
@@ -158,9 +163,14 @@ default MessageEditCallbackAction editComponents(@Nonnull Collection<? extends L
      *         The new message components, such as {@link ActionRow}
      *
      * @throws IllegalArgumentException
-     *         If the provided components are null
+     *         <ul>
+     *             <li>If any of the provided LayoutComponents is null</li>
+     *             <li>If any of the provided Components are not compatible with messages</li>
```

Comment:
Inconsistency here: "if any of the ... is/are"

Reply:
Thanks